.pre-commit-config.yaml.jinja

```diff
@@ -67,14 +67,18 @@ repos:
     rev: {{ repo_rev.black }}
     hooks:
       - id: black
-  - repo: https://github.com/prettier/prettier
+  - repo: https://github.com/prettier/pre-commit
     rev: {{ repo_rev.prettier }}
     hooks:
       - id: prettier
         name: prettier + plugin-xml
         additional_dependencies:
+          - prettier@2.1.2
           - "@prettier/plugin-xml@{{ repo_rev.prettier_xml }}"
         args:
+          - --write
+          - --list-different
+          - --ignore-unknown
```

Comment:
The problem here is that we don't know what will be the outcome of https://github.com/prettier/pre-commit/pull/18 and https://github.com/prettier/pre-commit/pull/17, so maybe it's better to stick to just changing language version and let these things evolve a bit before we do this change.